### DISCUSSION TITLE
How to load local .crx file in Cefsharp.

### DISCUSSION THREAD

**Discussion by songminkyu** [cefsharp/CefSharp#3592] [General]
I'm currently using CefSharp in WPF.
So I want to load a .crx file via CefSharp.
I'd appreciate it if you could give me a specific way.

**Comment by songminkyu** [comment]
The target file to be applied is attached.
[1.3_0.zip](https://github.com/cefsharp/CefSharp/files/6575134/1.3_0.zip)

**Reply by amaitland** [reply]
Please don't use the feature request template to ask questions. Please use https://github.com/cefsharp/CefSharp/discussions in future. 

See https://github.com/cefsharp/CefSharp/issues/2497 for details on extension support. You need to extract your crx to disk first (crx is just a zip file)

CEF only supports a subset of the chromium extension API as detailed in https://bitbucket.org/chromiumembedded/cef/issues/1947/add-support-for-chrome-extensions

Confirm CEF has support for the extension APIs before continuing.
